<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- Locationの設定＆マップの読み込み -->
  <arg name="map_name" default="rcap_2021_map"/>
  <rosparam file="$(find happymimi_params)/location/$(arg map_name).yaml" command="load" ns="/location" />
  <node name="map_server" pkg="map_server" type="map_server" args="$(find happymimi_navigation)/maps/$(arg map_name).yaml" />

  <!-- AMCL -->
  <include file="$(find happymimi_navigation)/launch/include/amcl.launch.xml"/>

  <!-- Move base -->
  <include file="$(find happymimi_navigation)/launch/include/move_base.launch.xml"/>

  <!-- navi location server -->
  <node pkg="happymimi_navigation" type="navi_location.py" name="navi_location_server" output="screen" />
  <node pkg="happymimi_navigation" type="navi_coord.py" name="navi_coord_server" output="screen" />

</launch>
